pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/UserUtil.java

```diff
@@ -254,6 +254,10 @@ private static void getList(Connection conn, String query, String role, String u
         }
     }
 
+    public boolean userHasRole(String realm, String userId, String role, String userIdRequest) {
```

Comment:
Why not go full bore and just rename the underlying method?  Happy to put a thumb on that as a separate PR.

Reply:
'checkUserAccess` is used in many places, so did not want to expand the number of file in the PR.
Although I think `userHasRole` may be more accurate, the method does a bit more checking on input. 
The motivation for changing was to avoid a collision with the existing method name, not to provide a better name, so we would need to resolve that issue to make the change you suggest.